<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyRouter</class>
 <widget class="QMainWindow" name="PyRouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyRouter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Forwarding_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Forwarding Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="ARPCacheTablelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP Cache Table</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>249</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QPushButton" name="ARPInsertButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ARPViewButton">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>MAC Destination</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="MACDestinationEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP Destination</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="IPDestinationEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Table Number</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ARPTableNumberEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="ForwardingTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Destination IP</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Netmask</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gateway IP</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Flag</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Interface</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Metric</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="ARPCacheTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>19</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>IP Address</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC Address</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string> Table Number</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>265</width>
      <height>253</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1" colspan="3">
        <widget class="QLineEdit" name="ForwardingMetricEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ForwardingFlagLable">
         <property name="text">
          <string>Flag</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="ForwardingInterfaceEdit"/>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="HostCheckBox">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ForwardingMetricLable">
         <property name="text">
          <string>Metric</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="GatewayCheckBox">
         <property name="text">
          <string>Gateway</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ForwardingGatewaylabel">
         <property name="text">
          <string>Gateway</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ForwardingInterfaceLable">
         <property name="text">
          <string>Interface</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="ForwardingNetmaskEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ForwardingDestinationlabel">
         <property name="text">
          <string>Destination</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="ForwardingGatewayEdit"/>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="ForwardingDestinationEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ForwardingNetmaskLable">
         <property name="text">
          <string>Netmask</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="UpCheckBox">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="ForwardingViewButton">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="ForwardingInsertButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
